concepticondata/conceptlists/Lundgren-2020-275.tsv

```diff
@@ -0,0 +1,276 @@
+Lundgren-2020-275-117	117	117	GOURD	411	GOURD
+Lundgren-2020-275-118	118	118	GRAB	855	SEIZE
+Lundgren-2020-275-119	119	119	GRANDCHILD (F. EGO)		
+Lundgren-2020-275-120	120	120	GRANDFATHER	1383	GRANDFATHER
+Lundgren-2020-275-121	121	121	GRASS	606	GRASS
+Lundgren-2020-275-122	122	122	GREEN	1425	GREEN
+Lundgren-2020-275-123	123	123	GRILL		
+Lundgren-2020-275-124	124	124	HAMMOCK	2019	HAMMOCK
+Lundgren-2020-275-125	125	125	HAND	1277	HAND
+Lundgren-2020-275-126	126	126	HAPPY	1495	HAPPY
+Lundgren-2020-275-127	127	127	HARD	1884	HARD
+Lundgren-2020-275-128	128	128	HEAD	1256	HEAD
+Lundgren-2020-275-129	129	129	HEAR (TR.)	1408	HEAR
+Lundgren-2020-275-130	130	130	HEART	1223	HEART
+Lundgren-2020-275-131	131	131	HEAVY	1210	HEAVY
+Lundgren-2020-275-132	132	132	HIGH UP (ADV.)	1265	HIGH
+Lundgren-2020-275-133	133	133	HOLE	1667	HOLE
+Lundgren-2020-275-134	134	134	HORN	1393	HORN (ANATOMY)
+Lundgren-2020-275-135	135	135	HORSEFLY	3262	HORSEFLY
+Lundgren-2020-275-136	136	136	HOUSE	1252	HOUSE
+Lundgren-2020-275-137	137	137	HOWLER MONKEY	433	HOWLER MONKEY
+Lundgren-2020-275-138	138	138	HUNGRY	1533	BE HUNGRY
+Lundgren-2020-275-139	139	139	HURT		
+Lundgren-2020-275-140	140	140	HUSBAND	1200	HUSBAND
+Lundgren-2020-275-141	141	141	INDIGENOUS PERSON	3612	INDIGENOUS PEOPLES OF THE AMERICAS
+Lundgren-2020-275-142	142	142	INVITE (TR.)	1611	INVITE
+Lundgren-2020-275-143	143	143	ISLAND	622	ISLAND
+Lundgren-2020-275-144	144	144	JOIN~ATTACH	1917	TIE
+Lundgren-2020-275-145	145	145	KILL	1417	KILL
+Lundgren-2020-275-146	146	146	KNIFE	1352	KNIFE
+Lundgren-2020-275-147	147	147	LAKE	624	LAKE
+Lundgren-2020-275-148	148	148	LAND	626	LAND
+Lundgren-2020-275-149	149	149	LARVA	2640	LARVA
+Lundgren-2020-275-150	150	150	LATE	477	LATE
+Lundgren-2020-275-151	151	151	LEAVE (TR.)		
+Lundgren-2020-275-152	152	152	LEAVE FROM (INTR.)	1752	LEAVE
+Lundgren-2020-275-153	153	153	LIVER~HEART		
+Lundgren-2020-275-154	154	154	LONG	1203	LONG
+Lundgren-2020-275-155	155	155	LONG TIME AGO	373	LONG AGO
+Lundgren-2020-275-156	156	156	LOOK (INTR.)	1819	LOOK
+Lundgren-2020-275-157	157	157	LOOK FOR (TR.)	1468	LOOK FOR
+Lundgren-2020-275-158	158	158	LOSE WAY		
+Lundgren-2020-275-159	159	159	LOSE ONESELF		
+Lundgren-2020-275-160	160	160	LOUSE	1392	LOUSE
+Lundgren-2020-275-161	161	161	LUNG~BREATHE (INTR.)		
+Lundgren-2020-275-162	162	162	MACAW	1630	MACAW
+Lundgren-2020-275-163	163	163	MANIOC FLOUR	324	MANIOC FLOUR
+Lundgren-2020-275-164	164	164	MANY~MUCH	1198	MANY
+Lundgren-2020-275-165	165	165	MIX (TR.)~PRESS		
+Lundgren-2020-275-166	166	166	MONKEY	1350	MONKEY
+Lundgren-2020-275-167	167	167	MOON	1313	MOON
+Lundgren-2020-275-168	168	168	MOSQUITO	1509	MOSQUITO
+Lundgren-2020-275-169	169	169	MOURN	1955	GRIEVE
+Lundgren-2020-275-170	170	170	MOUSE~RAT	2139	MUROID (MOUSE OR RAT)
+Lundgren-2020-275-171	171	171	MOUTH	674	MOUTH
+Lundgren-2020-275-172	172	172	NAIL (BODY PART)	1896	FINGERNAIL OR TOENAIL
+Lundgren-2020-275-173	173	173	NAME		
+Lundgren-2020-275-174	174	174	NECK~THROAT		
```

Comment:
Hm, NECK refers to the whole Hals, both NAPE OF NECK and THROAT, so having a concept NECK OR THROAT that refers to both the whole and a part seems a bit strange (if not unheard of).

Reply:
If there's no neck in the data, link to neck.